src/main/java/org/dataone/bookkeeper/resources/UsagesResource.java

```diff
@@ -203,14 +203,17 @@ public UsagesResource(Jdbi database, DataONEAuthHelper dataoneAuthHelper) {
     public Usage create(
             @Context SecurityContext context,
             @NotNull @Valid Usage usage) throws WebApplicationException {
+        String DEFAULT_STATUS = "active";
         // Insert the usage after it is validated
-
         // The calling user injected in the security context via authentication
         Customer caller = (Customer) context.getUserPrincipal();
         boolean isAdmin = this.dataoneAuthHelper.isAdmin(caller.getSubject());
 
         if ( isAdmin ) {
             try {
+                // On create, status is always set to 'active'
+                if(usage.getStatus().compareToIgnoreCase(DEFAULT_STATUS) != 0)
+                    usage.setStatus(DEFAULT_STATUS);
```

Comment:
Minor code formatting:
- Please use code blocks
- Please give `if` statements whitespace
```suggestion
                if (usage.getStatus().compareToIgnoreCase(DEFAULT_STATUS) != 0) {
                    usage.setStatus(DEFAULT_STATUS);
                }
```